ammo-fe/src/App.tsx

```diff
@@ -1,20 +1,49 @@
-import React, { ReactElement } from 'react';
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { RootReducer } from 'redux/reducers';
+import useWs from 'hook/useWs';
+import { Bullet } from 'shared/typings/Bullet';
+
+function App(): React.ReactElement {
+    useWs();
+
+    const connected = useSelector((state: RootReducer) => state.ws.connected);
```

Comment:
I _could_ get why 'connected' needs to have some links with the ws, but bullets don't. They do not need to be prefixed